api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -325,6 +331,9 @@ type NicClusterPolicySpec struct {
 	// DOCATelemetryService exposes telemetry from NVIDIA networking components to prometheus.
 	// See: https://docs.nvidia.com/doca/sdk/nvidia+doca+telemetry+service+guide/index.html
 	DOCATelemetryService *DOCATelemetryServiceSpec `json:"docaTelemetryService,omitempty"`
+	// DSpectrumXOperator exposes NVIDIA Spectrum-X Operator.
```

Comment:
Typo in the comment `DSpectrumXOperator `

Reply:
fixed